### PR TITLE
Footer changes

### PR DESCRIPTION
**Pull request by mcspach**
Addresses #148 by fixing positioning and content on the footer element.
please see comment/images on issue for details.

### CODE REVIEW

**Comment by mcspach** [comment]
The last commit addresses spacing issues mentioned in Wed meeting, changes path for icon imgs, and adds all links to social.
Should be ready for merge.

File: _includes/footer.html [hunk 1]
```diff
@@ -1,18 +1,22 @@
 <!-- This will be our footer -->
-<footer class="footer mdl-mega-footer">
-  <div class="mdl-mega-footer__bottom-section">
-    <div class="mdl-mega-footer__drop-down-section">
-      <div class="mdl-mega-footer__heading" >Hack For LA</div>
-      <p class="footer_slogan">100 Automations is one of the many projects at Hack for LA, Code for America’s Los
-        Angeles chapter.
+<footer class="footer mdl-mini-footer">
```
Comment (drubgrubby) [inline _includes/footer.html]: This references a .footer class that doesn't seem to exist anywhere.  (Unless I'm not seeing it.)  If it doesn't exist, I'd recommend removing the reference to it.

File: _includes/footer.html [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+  <div class="footer-slogan">
+      <p class="">100 Automations is one of the many projects at Hack&nbsp;for&nbsp;LA, Code&nbsp;for&nbsp;America’s Los&nbsp;Angeles&nbsp;chapter.
```
Comment (drubgrubby) [inline _includes/footer.html:4]: Very clever to use html entities to keep the names from breaking. I'm going to steal that.

File: _sass/includes/footer.scss [hunk 3]
```diff
@@ -1,7 +1,32 @@
...
-}
+    p {
+      margin: 0;
+      font-size: 0.85rem;
+      line-height: 120%;
+      margin: 20px 16px 20px 0;
+    }
+  }
+  // MDL social links customizations
+  .mdl-mini-footer__link-list {
```
Comment (drubgrubby) [inline _sass/includes/footer.scss:27]: This is super-minor, but I mention it nonetheless. 

When you cross over the 760px width threshold, the line of text jumps up or down (depending on which direction you are going) just a bit, like one line-height. It seem to be that the flex boxes surrounding the icons changes size, which pushes the text up or drops it down.

Screen Shot 2021-02-12 at 10 38 21 AM
Screen Shot 2021-02-12 at 10 38 31 AM

You might try constraining the height of the flexbox, or something similar, to keep this from happening.  As I said, super minor, but if I noticed it, someone else will.

**Review by drubgrubby — CHANGES_REQUESTED** [review]
@mcspach - This looks great!  I've suggested a couple of nerdy, minor changes, but really nice work.

**Comment by mcspach** [comment]
Screen Shot 2021-02-12 at 3 08 34 PM
Screen Shot 2021-02-12 at 3 08 54 PM

^^ Above is img from desktop and mobile ^^
-I am unsure about the small padding change that happens at 760px. The change is very small, but it may have to to with the Material Design classes.
-This last commit will get rid of the unused 'footer' class.

**Comment by daniellex0** [comment]
The CSS looks good to me 👍

**Comment by mdothman** [comment]
I've looked over the code and think you did a spectacular job!

**Comment by mdothman** [comment]
Your code looks spectacular, great job. Fixes issue #148

**Pull request merged**